index.html

```diff
@@ -17,15 +17,25 @@
     <div class="Main-container">
         <div class="container">
             <header class="header hoverable">
-                <a>My Logo</a>
-                <nav><img src="/photos/Union.png" alt="Hamburger Icon"></nav>
+                <h1 class="gol"><a>My Logo</a></h1>
+
+                <nav>
+                    <div class="hidden">
+                        <ul class="hidden-ul">
+                            <li class="po"><a href="#">Portoflio</a></li>
+                            <li class="ab"><a href="#">About</a></li>
+                            <li class="co"><a href="#">Contact</a></li>
+                        </ul>
+                    </div>
```

Comment:
- [ ] Your portfolio should be responsive in all devices but I noticed that  there is a gap space in 670px width screens, so it would be better if you fix it in order your portfolio become responsive in all devices
![image](https://user-images.githubusercontent.com/61790587/167229494-7a72cc11-3649-4dce-bf79-68c97f37dd3b.png)